code/modules/gear_presets/cmb.dm

```diff
@@ -0,0 +1,693 @@
+	name = "CMB - Interstellar Human Rights Observer"
+	paygrade = "GS-3"
+	idtype = /obj/item/card/id/lanyard
+	role_comm_title = "OBS"
+	flags = EQUIPMENT_PRESET_EXTRA
+
+	assignment = "Interstellar Human Rights Observer"
+	rank = JOB_CMB_OBS
+	skills = /datum/skills/civilian/survivor/doctor
+	languages = list(LANGUAGE_ENGLISH, LANGUAGE_SPANISH, LANGUAGE_RUSSIAN)
+	faction = FACTION_CMB
+	faction_group = list(FACTION_LIST_MARINE_CMB)
+
+/datum/equipment_preset/CMB/observer/load_gear(mob/living/carbon/human/H)
+
+	//clothes
+	H.equip_to_slot_or_del(new headset_type, WEAR_L_EAR)
+	H.equip_to_slot_or_del(new /obj/item/device/flashlight/pen, WEAR_R_EAR)
+	H.equip_to_slot_or_del(new /obj/item/clothing/under/liaison_suit/suspenders, WEAR_BODY)
+	H.equip_to_slot_or_del(new /obj/item/clothing/accessory/health/ceramic_plate, WEAR_ACCESSORY)
+	H.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/vest, WEAR_JACKET)
+	H.equip_to_slot_or_del(new /obj/item/device/flashlight, WEAR_J_STORE)
+	H.equip_to_slot_or_del(new /obj/item/storage/belt/medical/lifesaver/full, WEAR_WAIST)
+	//holding
+	H.equip_to_slot_or_del(new /obj/item/clothing/gloves/marine/insulated, WEAR_HANDS)
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine/knife, WEAR_FEET)
+	H.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/sensor, WEAR_EYES)
+	H.equip_to_slot_or_del(new /obj/item/clothing/mask/surgical, WEAR_FACE)
+	H.equip_to_slot_or_del(new /obj/item/device/healthanalyzer, WEAR_L_HAND)
+	H.equip_to_slot_or_del(new /obj/item/device/camera, WEAR_R_HAND)
+	//pouches
+	H.equip_to_slot_or_del(new /obj/item/storage/pouch/autoinjector/full, WEAR_L_STORE)
+	H.equip_to_slot_or_del(new /obj/item/storage/pouch/general/large, WEAR_R_STORE)
+	H.equip_to_slot_or_del(new /obj/item/tool/extinguisher/mini, WEAR_IN_R_STORE)
+	H.equip_to_slot_or_del(new /obj/item/roller, WEAR_IN_R_STORE)
+	H.equip_to_slot_or_del(new /obj/item/reagent_container/spray/pepper, WEAR_IN_R_STORE)
+	//backpack and stuff in it
+	H.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel/lockable/liaison, WEAR_BACK)
+	H.equip_to_slot_or_del(new /obj/item/device/camera_film, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/device/binoculars, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/pen/clicky, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/paper/carbon, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/folder, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/device/radio, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/clipboard, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/device/taperecorder, WEAR_IN_BACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/crowbar, WEAR_IN_BACK)
+
+
+//############ Anchorpoint Station Colonial Marines - They serve as reinforcements for the Colonial Marshals of Anchorpoint Station. #############
+//Anchorpoint Station Squad Marine - Similar to the Movie squad but nerfed a bit.
+
+/datum/equipment_preset/uscm/CMB
```

Comment:
These may benefit from pulling directly from their USCM counterparts before declaring a subtype, rather than declaring CMB subtype before things like /leader, as this will reduce the amount of duplicate code